registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/service/impl/RegistrationStatusServiceImpl.java

```diff
@@ -401,27 +408,63 @@ public void updateRegistrationStatus(InternalRegistrationStatusDto registrationS
 	/**
 	 * Convert entity to dto and get external status.
 	 *
-	 * @param entity
-	 *            the entity
+	 * @param entities
+	 *            the entities
 	 * @return the registration status dto
 	 */
-	private RegistrationStatusDto convertEntityToDtoAndGetExternalStatus(RegistrationStatusEntity entity) {
+	private RegistrationStatusDto convertEntityToDtoAndGetExternalStatus(List<RegistrationStatusEntity> entities) {
```

Comment:
Please don't change the existing method, since /search api will be calling this methods and our code should be backward compatible

Reply:
it is backward compatible...i have also tested /search api...

Reply:
created separate method for external status api...